### DISCUSSION TITLE
写了一个获取群成员的function，但是没有实现求大神帮忙指导一下

### DISCUSSION THREAD

**Discussion by yixianbaowang** [MR-XieXuan/MiraiTravel#10] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/125167113/219244700-a4bc00b7-5b93-45a6-85cf-03f4c61dd8cd.png)
function get_group_members($sessionKey = "", $target, $other = array())
{
    $content = array();
    if (!empty($sessionKey)) {
        $content['sessionKey'] = (string)$sessionKey;
    }
    $content['target'] = (int)$target;

    $funcName = basename(str_replace('\\', '/', __FUNCTION__));
    return adapter_manager("auto", $funcName, $content, $other);
}

![image](https://user-images.githubusercontent.com/125167113/219244986-1a18b011-6c8f-4ada-8b27-f5b94c8c620d.png)
function get_group_members($target, $other = array())
    {
        $logSystem = new LogSystem($this->get_qq(), "QQBot");
        $logSystem->write_log("GroupManagement", "get_group_members", "$target get_group_members" . " for " . $this->get_session_key());
        return get_group_members(
            $this->get_session_key(),
            $target,
            array("qqbot" => $this)
        );
    }
执行的代码
![image](https://user-images.githubusercontent.com/125167113/219245208-93a35784-2e3a-434e-8976-a11472ec0373.png)

**Comment by MR-XieXuan** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/95032548/219313736-72f6a900-e9aa-4c1f-b54e-7186a57d6780.png)
GET请求调用 API 分配器需要传入HTTP请求类型,如下：
![image](https://user-images.githubusercontent.com/95032548/219313941-5f21fc07-d1a5-4eca-9469-1686c836b4df.png)
j加上 $other['apiType'] = "GET"
